Sprint-2/errors/0.js

```diff
@@ -3,7 +3,17 @@
 // call the function capitalise with a string input
 // interpret the error message and figure out why an error is occurring
 
+// function capitalise(str) {
+//   let str = `${str[0].toUpperCase()}${str.slice(1)}`;
+//   return str;
+// }
+
+//correction 
+// the problem is  the capitalise function is that it is re declaring the str parameter inside the function using let.
+// we canot do that cause the str is already a defined function.
```

Comment:
It's not a defined function. It is defined, but what is it, exactly? Try to be precise here in your explanation.